src/main/_docs/admin-guide/admin-cli.md

```diff
@@ -27,7 +78,7 @@ Codenvy depends upon Docker images. We use Docker images in three ways:
 
 You can control the nature of how Codenvy downloads these images with command line options. All image downloads are performed with `docker pull`.
 
-| Mode | Description |
+| Mode>>>>> | Description |
```

Comment:
Yes, for the moment it's the most reliable way to force markdown to size a column to the width you want. Sometimes adding "&nbsp;" works but not in all cases. Otherwise it make some strange decisions about which columns should be big or small.